docs/components/forms.md

```diff
@@ -889,6 +883,20 @@ Custom `<select>` menus need only a custom class, `.custom-select` to trigger th
 
 Custom selects degrade nicely in IE9, receiving only a handful of overrides to remove the custom `background-image`. **Multiple selects (e.g., `<select multiple>`) are not currently supported.**
 
+### Color picker
+
+{% callout danger %}
+#### Internet Explorer compatibility
+
+While Bootstrap supports styling `<input type="color">` elements, Internet Explorer Edge 13 and below don’t support. Use custom JavaScript to handle it in these browsers.
```

Comment:
Per Can I Use, the full list of relevant non-supporting browsers is:
IE, Edge 13, OS X Safari 9.1, iOS 9.3 Safari